benchmark/src/lib.rs

```diff
@@ -73,7 +73,12 @@ pub struct Benchmarker {
     /// Use WalletLibrary to setup the benchmarking environment.
     wallet: WalletLibrary,
     /// Interface to metric counters in validator nodes, e.g., #commited_txns in storage.
+    /// While it is not needed anymore as we are waiting on accounts' sequence numbers,
```

Comment:
Reply to @fbandersnatch : Right now in `submit_txns`, with `debug_client` we figure out how many TXNs are committed during submitting TXNs.

This info is not super important: it is not used anywhere except appearing in a log msg that summarizing the submission process. But I think this is still a reason to keep `debug_client`. Maybe in a later diff we can remove this entirely with the cmdline argument.